sys/mips/timer.c

```diff
@@ -100,14 +100,15 @@ static int mips_timer_stop(timer_t *tm) {
 static bintime_t mips_timer_gettime(timer_t *tm) {
   device_t *dev = tm->tm_priv;
   mips_timer_state_t *state = dev->state;
-  uint32_t sec, ticks;
+  uint64_t sec, ticks;
```

Comment:
Do we need to change ticks to `uint64_t` as well?

Reply:
Fixed!